src/azul/terraform.py

```diff
@@ -685,6 +685,11 @@ def tf_config(self, app_name):
 class VPC:
     num_zones = 2  # An ALB needs at least two availability zones
 
+    # To avoid brittle coupling between terraform resources and code which
+    # references those resources, the names of these resources are defined here.
+    default_vpc_name = 'default'  # For the aws_default_vpc
+    default_security_group_name = 'default'  # For the aws_default_security_group
```

Comment:
I didn't ask for a comment on why these attributes exist, which is usually to avoid brittleness when duplicates aren't in close proximity, or simply to avoid duplication. What you wrote distracts from the main point that I requested you clarify. I also wouldn't bake in details about the usages since that reintroduces brittleness.

```suggestion
    # These are TF resource names, the real-world resource names are fixed by AWS.
    default_vpc_name = 'default'
    default_security_group_name = 'default'
```